lib/openstudio/workflow/util/energyplus.rb

```diff
@@ -345,229 +348,229 @@ def self.merge_output_table_summary_reports(current_object, new_object)
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 11
-    WaterSystems:DistrictHeating,!- Variable or Meter 12 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 12
-    Cogeneration:DistrictHeating,!- Variable or Meter 13 Name
-    ValueWhenMaximumOrMinimum;            !- Aggregation Type for Variable or Meter 13
-
-Output:Table:Monthly,
-  Building Energy Performance - District Cooling Peak Demand,  !- Name
-    2,                       !- Digits After Decimal
-    DistrictCooling:Facility,  !- Variable or Meter 1 Name
-    Maximum,            !- Aggregation Type for Variable or Meter 1
-    InteriorLights:DistrictCooling,  !- Variable or Meter 1 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 1
-    ExteriorLights:DistrictCooling,  !- Variable or Meter 2 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 2
-    InteriorEquipment:DistrictCooling,  !- Variable or Meter 3 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 3
-    ExteriorEquipment:DistrictCooling,  !- Variable or Meter 4 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 4
-    Fans:DistrictCooling,        !- Variable or Meter 5 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 5
-    Pumps:DistrictCooling,       !- Variable or Meter 6 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 6
-    Heating:DistrictCooling,     !- Variable or Meter 7 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 7
-    Cooling:DistrictCooling,     !- Variable or Meter 8 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 8
-    HeatRejection:DistrictCooling,  !- Variable or Meter 9 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 9
-    Humidifier:DistrictCooling,  !- Variable or Meter 10 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 10
-    HeatRecovery:DistrictCooling,!- Variable or Meter 11 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 11
-    WaterSystems:DistrictCooling,!- Variable or Meter 12 Name
-    ValueWhenMaximumOrMinimum,            !- Aggregation Type for Variable or Meter 12
-    Cogeneration:DistrictCooling,!- Variable or Meter 13 Name
-    ValueWhenMaximumOrMinimum;            !- Aggregation Type for Variable or Meter 13
- HEREDOC
+          <<~HEREDOC
+            Output:Table:Monthly,
+              Building Energy Performance - Electricity,  !- Name
+                2,                       !- Digits After Decimal
+                InteriorLights:Electricity,  !- Variable or Meter 1 Name
+                SumOrAverage,            !- Aggregation Type for Variable or Meter 1
```

Comment:
Not sure I like this indentation change, but I don't like the extra leading spaces will have any effect (that is: end up in the IDF file) so it's probably fine.

Reply:
Now I get it, you use `<<~HEREDOC` not `<<-HEREDOC` so it ignores the leading whitespace.